config/dev/dgutman-dsa-server-preprod-ec2.yaml

```diff
@@ -0,0 +1,27 @@
+# This stack is based on `config/prod/dgutman-htan-new2-ec2.yaml`
+# in https://github.com/Sage-Bionetworks/sandbox-provisioner/
+template_path: "remote/managed-ec2-linux-v2.j2"
```

Comment:
i suggest you don't use this template since it's sorta specific to sandbox and scicomp.  You can either create a custom and put it in this repo or create a more generic one and put it in aws-infra repo to share.

Reply:
@zaro0508 would the template in this PR be sufficient provided we are spinning a sandbox instance? I.e. what are the advantages of a custom or generic templates over the one here?

Reply:
I think it's fine to use this template... will also want the other sandbox infrastructure in this account, like maintenance window, quarantine security group, etc.

Reply:
Is this "sandbox-like infrastructure" a requirement for this pre-production server? Or can it wait until the production deployment? This is the first time that I hear about these additional requirements for new AWS accounts and would have expected these to be deployed automatically by org-formation if they're even required on dev accounts.

Reply:
Yes, I agree that it should be deployed for org-f or sceptre for all member accounts. It shouldn't require any extra set up